.github/workflows/ci.yml

```diff
@@ -97,14 +97,3 @@ jobs:
   #     - run: npx lerna publish from-package --yes --no-verify-access --canary
   #     - if: always()
   #       run: rm .npmrc
-
-  docs:
```

Comment:
@jacqueswho actually -- we actually want to keep this `docs` pipeline in. This pipeline doesn't deploy docs, it just builds the docs!

We currently deploy docs using readthedocs -- https://docs.readthedocs.io/en/stable/versions.html -- it looks like the solution here is to use the `stable` version of the docs site (If your project has any tags or branches with a name following [semantic versioning](https://semver.org/), we also create a stable version, tracking your most recent release. If you want a custom stable version, create either a tag or branch in your project with that name.).

Let's discuss more on Friday.

Reply:
To be able to merge this, maybe make the `docs` step be a no-op, or perhaps just leave it as is since having the ability to view the latest docs could be useful...

Reply:
@jacqueswho Just revert this and you are good to go!